docs/src/main/asciidoc/spring-cloud-gateway.adoc

```diff
@@ -775,6 +775,30 @@ spring:
 
 NOTE: At this time a URI using the `forward` protocol does not support using the retry filter.
 
+=== RequestSize GatewayFilter Factory
```

Comment:
The javadoc for the filter says the default size is 5MB this would be useful to include in the documentation